test/memory/sungeun/refCount/domainMaps.lm-numa.good

```diff
@@ -77,4 +77,4 @@ Create domain and array:
 	0 bytes leaked
 	0 private objects leaked
 total:
-	1104 bytes leaked
+	0 bytes leaked
```

Comment:
Hmm, good question. I'd be inclined to merge and let nightly testing notify us if things changed.